src/actions/contract.js

```diff
@@ -21,40 +21,48 @@
 import fs from 'fs-extra';
 import path from 'path';
 import { encode } from '@aeternity/aepp-sdk';
+import { utils } from '@aeternity/aeproject';
 import { initSdk, initSdkByWalletFile } from '../utils/cli';
 import { print, printTransaction, printUnderscored } from '../utils/print';
 import CliError from '../utils/CliError';
 
 const resolve = (filename) => path.resolve(process.cwd(), filename);
 
-// ## Function which compile your `source` code
-export async function compile(filename, options) {
-  const sdk = await initSdk(options);
-  const { bytecode } = await sdk.compilerApi.compileContract({
-    code: (await fs.readFile(resolve(filename))).toString(), options: {},
-  });
-  if (options.json) print({ bytecode });
-  else print(`Contract bytecode: ${bytecode}`);
-}
-
 async function getContractParams({
   descrPath, contractAddress, contractSource, contractBytecode, contractAci,
 }, { dummySource, descrMayNotExist } = {}) {
   let descriptor = {};
   if (descrPath && (!descrMayNotExist || await fs.exists(resolve(descrPath)))) {
     descriptor = await fs.readJson(resolve(descrPath));
   }
+  if (contractSource) {
+    const contractSourcePath = resolve(contractSource);
+    descriptor.source = (await fs.readFile(contractSourcePath)).toString();
+    // TODO: remove after fixing https://github.com/aeternity/aeproject/issues/435
```

Comment:
maybe you can elaborate the need, maybe I missunderstood the "debug output" nature

Reply:
In this case, with `--json` argument cli's output should be a valid json object. If I use `utils.getFilesystem` as is then I get json object prefixed with "Adding include to filesystem" message that can't be parsed as json.